config/routes.rb

```diff
@@ -11,6 +11,8 @@
     end
   end
 
+  resources :organizations
```

Comment:
Make sure to include some `:only` parameters here: don't yet want people to be able to delete organizations.